release-engineering/README.md

```diff
@@ -203,7 +203,7 @@ that any final cherry-pick content is merged ahead of that last release.
   - CRI tools
   - CNI plugins
   - OS base images (example: debian-base)
-  - debian-iptables
+  - distroless-iptables
   - debian-hyperkube-base
```

Comment:
I think this image doesn't exist for a long time, should we remove it from this doc?